pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -2,34 +2,46 @@ package bootstrappolicy
 
 import (
 	"reflect"
+	"sort"
 	"testing"
 
 	"k8s.io/apiserver/pkg/authentication/serviceaccount"
 
 	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	scc "github.com/openshift/origin/pkg/security/securitycontextconstraints"
 )
 
 func TestBootstrappedConstraints(t *testing.T) {
-	expectedConstraints := []string{
-		SecurityContextConstraintPrivileged,
+	// ordering of expectedConstraintNames is important, we check it against scc.ByPriority
+	expectedConstraintNames := []string{
+		SecurityContextConstraintsAnyUID,
+		SecurityContextConstraintsHostNetwork,
 		SecurityContextConstraintRestricted,
 		SecurityContextConstraintNonRoot,
-		SecurityContextConstraintHostMountAndAnyUID,
 		SecurityContextConstraintHostNS,
-		SecurityContextConstraintsAnyUID,
-		SecurityContextConstraintsHostNetwork,
+		SecurityContextConstraintHostMountAndAnyUID,
+		SecurityContextConstraintPrivileged,
```

Comment:
@pweil- Do you agree with such order?

I don't understand it: if user has access to all the SCCs then I'd expect the following order: `anyuid`, `restricted`, ..., `privileged`.

In other words, the question why `hostnetwork` goes before `restricted`?